generator/views/events_view.py

```diff
@@ -76,11 +76,9 @@ def to_lookml(self, bq_client, v1_name: Optional[str]) -> Dict[str, Any]:
 
         # set document_id as primary key if it exists in the underlying table
         # this will allow one_to_many joins
-        document_id_field = self.get_document_id(dimensions, "events")
-        if document_id_field is not None:
-            view_defn["dimensions"] = [
-                {"name": document_id_field, "primary_key": "yes"}
-            ]
```

Comment:
Would this remove `document_id` from the views/explores? Should we still keep the field around in case any existing looks or dashboards are referencing it?

Reply:
I'll confirm but I don't think it removes it. It should still be present in the underlying `_table` view that this extends.

Reply:
It's defined here: https://github.com/mozilla/looker-hub/blob/main/firefox_desktop/views/events_stream_table.view.lkml#L116-L119